tutorials/remote-control/index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "LEGO Remote Control"
+image:
+    local: "lego-remote.jpg"
+layout: theme
+description: "Theme description"
```

Comment:
Looks like template text that needs to be replaced.

Reply:
Yes, but please advise how these category md files are supposed to work, I could not figure it all out (and don't know how to preview a change) , so I just tried to copy others (which had left "Theme description" in). This description text seems to be ignored, and it gets some text and image from somewhere that I can't see in the tree. So I don't know if my image will work either, etc.

Reply:
The website repo that this plugs in to is currently private, so I'm afraid there isn't an easy way to preview right now.

If all of the other themes have the same text, then I guess we don't need to worry about it.

Reply:
Indeed, you can leave the theme names for now. It would be nice to add a tutorial on how to add a tutorial (or a project).